File: Practice/13/C++/example13/example13/example13.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main()
{
    setlocale(LC_ALL, "Russian");

    int n, resultat;

  

        n = 0;
        cin >> n;
        if (n >= 2 and n <= pow(10, 9)) {


            resultat = 0;
            for (int i = 2; i < 10; i++)
            {
                if (n % i == 0) {
                    resultat = 1;
                }


            }

            if (resultat and n != 2 and n != 3 and n != 5 and n != 7) {
                cout << "Составное";
            }
            else {
                cout << "Простое";
            }





        }
        else {
            cout << "Число n не соответствует условию"<<endl;
        }




    
}

File: Practice/13/Python/example13.py
n = int(input())
if (n >= 2 and n <= 10**9):
    resultat = 0
    i = 2
    while i<10:
        if n%i == 0:
            resultat = 1
        i +=1
    if (resultat and n != 2 and n != 3 and n != 5 and n != 7):
        print("Составное")
    else:
        print("Простое")
else:
    print("Число n не соответствует условию")